### PR TITLE
Jc/v/fac 28 set up vuzix app

### PR DESCRIPTION
**Pull request by carrolljody**

### CODE REVIEW

File: Vuzix/app/src/main/java/com/facilitation/view/DynamicThemeReceiver.kt [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+import android.appwidget.AppWidgetManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+
+class DynamicThemeReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context, intent: Intent) {
```
Comment (SkyKalazar) [inline Vuzix/app/src/main/java/com/facilitation/view/DynamicThemeReceiver.kt:11]: Do you actually need intent parameter here? It does not seem it is being used

File: Vuzix/app/src/main/java/com/facilitation/view/MainActivity.kt [hunk 2]
```diff
@@ -1,46 +1,106 @@
...
+        }
+        SpotifyMenuItem?.isEnabled = false
+    }
+
+    //Action Menu Click events
+    //This events where register via the XML for the menu definitions.
+    fun showHello(item: MenuItem?) {
+        val broadcastIntent = Intent("com.facilitation.view.GET")
+        broadcastIntent.putExtra("broadcastMessage", "Goodbye World!")
+        sendBroadcast(broadcastIntent)
```
Comment (SkyKalazar) [inline Vuzix/app/src/main/java/com/facilitation/view/MainActivity.kt:52]: Is this being sent when you get the GET action? If so, should I receive it?

File: Vuzix/app/src/androidTest/java/com/facilitation/view/ExampleInstrumentedTest.kt [hunk 3]
```diff
@@ -1,24 +0,0 @@
-package com.facilitation.view
```
Comment (SkyKalazar) [inline Vuzix/app/src/androidTest/java/com/facilitation/view/ExampleInstrumentedTest.kt:1]: Do you think there are some tests that could be written for this?

**Review by SkyKalazar — APPROVED** [review]

File: Vuzix/app/src/main/java/com/facilitation/view/MainActivity.kt (hunk 2 above)
Comment (carrolljody) [reply Vuzix/app/src/main/java/com/facilitation/view/MainActivity.kt:52]: this was for testing internal broadcasts before you broadcasted from the phone

File: Vuzix/app/src/main/java/com/facilitation/view/DynamicThemeReceiver.kt (hunk 1 above)
Comment (carrolljody) [reply Vuzix/app/src/main/java/com/facilitation/view/DynamicThemeReceiver.kt:11]: Not sure yet. It's from the template app

File: Vuzix/app/src/androidTest/java/com/facilitation/view/ExampleInstrumentedTest.kt (hunk 3 above)
Comment (carrolljody) [reply Vuzix/app/src/androidTest/java/com/facilitation/view/ExampleInstrumentedTest.kt:1]: I have one but might need to rearrange some formatting for the next task so I will finish the tests with that task

**Pull request merged**